tests/gtest/avif_fuzztest_helpers.cc

```diff
@@ -187,9 +187,20 @@ ::testing::Environment* SetEnv(const char* name, const char* value) {
 
 //------------------------------------------------------------------------------
 
+// Helper macros to ensure TEST_DATA_DIRS is a quoted string.
+// NOTE: TEST_DATA_DIRS MUST NOT be a quoted string before stringification
```

Comment:
`TEST_DATA_DIRS must NOT` is easier to read

Reply:
Done.